torch/csrc/jit/mobile/type_parser.cpp

```diff
@@ -37,6 +39,42 @@ TypeParser::TypeParser(std::string pythonStr)
   lex();
 }
 
+TypeParser::TypeParser(std::vector<std::string>& pythonStrs)
+    : start_(0), pythonStrs_(pythonStrs) {}
+
+// For the Python string list parsing, the order of the Python string matters.
+// In bytecode, the order of the type list correspondings to the order of
+// instruction. In nested type, the lowest level type will be at the beginning
```

Comment:
It would be great to order in export side, so that when importing, we don't worry about ordering and just simply parse the ordered types.

Reply:
Currently the export side orders the list correctly. This comment means if the list from export is shuffled, likely it won't be parsed correctly.